src/Localization.hs

```diff
@@ -86,6 +86,36 @@ myLocale "it" = TimeLocale {
                     time12Fmt = "%I:%M:%S %p"
                 }
 
+-- spanish locale date format
+-- contributed by conradolandia (https://github.com/conradolandia)
+myLocale "it" = TimeLocale {
```

Comment:
I guess it should be "es" or something instead of "it".

Languages must also be declared in Preprocessor.hs: langs = words "en fr it **es**"